src/bundle/ui-dev/src/modules/common/alert/alert.js

```diff
@@ -5,15 +5,22 @@ import { createCssClassNames } from '../helpers/css.class.names';
 
 const ICON_NAME_MAP = {
     info: 'system-information',
-    error: 'circle-close',
-    warning: 'warning-triangle',
-    success: 'checkbox',
+    error: 'notice',
+    warning: 'warning',
+    success: 'approved',
+};
+
+const ICON_SIZE_MAP = {
```

Comment:
considering it's perfect mapping in terms of content, maybe just use `.toLowerCase()`?